src/containers/home/components/loginModal/index.js

```diff
@@ -87,6 +87,7 @@ class LoginModal extends React.Component {
                         maxLength="90"
                         autoFocus
                       />
+                      <p style={{ marginTop: 0, marginBottom: 0, color: '#888888' }}>Pour LoL le nom d'utilisateur doit être le nom d'invocateur</p>
```

Comment:
Did you make sure the username validation regex matched the LoL regexp ?

Reply:
nope